test/click.spec.js

```diff
@@ -483,6 +483,10 @@ module.exports.describe = function({testRunner, expect, playwright, FFOX, CHROMI
       expect(await page.evaluate(() => window.result)).toBe('Was not clicked');
     });
 
+    it('should climb dom for pointer-events:none targets', async({page, server}) => {
+      await page.goto(server.PREFIX + '/input/pointer-events.html');
```

Comment:
nit: might as well do setContent for easier reading.

`await page.setContent('<button><label style="pointer-events:none">Click target</label></button>')`

Reply:
Done